### ISSUE TITLE
CSP errors in Chromium and Firefox

### ISSUE THREAD

**Issue by chigby** [freedomofpress/securedrop.org#781]
CSP Errors seen in production in browsers: Firefox v82, Chromium 87.0.4280.66

Representative errors below (some formatting applied for legibility)

```
Refused to apply inline style because it violates the following Content Security Policy directive:

"style-src 'self'
'sha256-kRJHclfjr7e5UYWHxtr0Bzdv2BiUtaSbDQe69HgEqXM='
'sha256-cMOfJ1K7bmWDFQ9IoI+B6fO37u9xMiBgP1rpm79IayM='
'sha256-Pf5JUUfhnnTVCCmSWFJ3qi/1j67vD2TeYvr7T6LxfqY='
'sha256-aJumNcjgS5IN0N559UWLFNCtnIIo3CqO862elt0w1A0='
'sha256-Rg1ua3eExI+in3cF/PWaHTHMjpiLQz/jTlIXr2kBY38='
'sha256-Zbh/ZO0Ff1YEynn0zSl56u5itxZmwkCVF3PgnnOm8u4='
'sha256-4ieA95gpQdpg9JDmuID1CQF8dJ/U0JnDqE4GQecAIdg='
'sha256-LAw02AamnUpPKuSLFUcg9Kh2SLuqSmaXiiV45Y21f84='".

Either the 'unsafe-inline' keyword, a hash ('sha256-kRJHclfjr7e5UYWHxtr0Bzdv2BiUtaSbDQe69HgEqXM='), or a nonce ('nonce-...') is required to enable inline execution.
```

```
Refused to apply inline style because it violates the following Content Security Policy directive:

"style-src 'self'
'sha256-kRJHclfjr7e5UYWHxtr0Bzdv2BiUtaSbDQe69HgEqXM='
'sha256-cMOfJ1K7bmWDFQ9IoI+B6fO37u9xMiBgP1rpm79IayM='
'sha256-Pf5JUUfhnnTVCCmSWFJ3qi/1j67vD2TeYvr7T6LxfqY='
'sha256-aJumNcjgS5IN0N559UWLFNCtnIIo3CqO862elt0w1A0='
'sha256-Rg1ua3eExI+in3cF/PWaHTHMjpiLQz/jTlIXr2kBY38='
'sha256-Zbh/ZO0Ff1YEynn0zSl56u5itxZmwkCVF3PgnnOm8u4='
'sha256-4ieA95gpQdpg9JDmuID1CQF8dJ/U0JnDqE4GQecAIdg='
'sha256-LAw02AamnUpPKuSLFUcg9Kh2SLuqSmaXiiV45Y21f84='".

Either the 'unsafe-inline' keyword, a hash ('sha256-cMOfJ1K7bmWDFQ9IoI+B6fO37u9xMiBgP1rpm79IayM='), or a nonce ('nonce-...') is required to enable inline execution.
```

**Comment by chigby** [comment]
Looks like these are coming from SVG tags in the logo:

```
<path d="M213.51,493.08V246.54L427,123.27V369.81ZM372.72,352.73a32.43,32.43,0,0,0,16.21-28.08v-96a16,16,0,0,0-24-13.87L267,271.38v142.4ZM300.29,290l48.57-28a7,7,0,0,1,10.43,6v46.32a18.8,18.8,0,0,1-9.61,16.4l-49.39,27.69Z" style="fill: #7683a9;fill-rule: evenodd" />
```

**Comment by chigby** [comment]
Potentially related: https://bugs.chromium.org/p/chromium/issues/detail?id=546106

**Comment by eloquence** [comment]
This has been fixed through recent SVG changes (#798).
